src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/JDTUtils.java

```diff
@@ -104,4 +104,17 @@ public static CodeAction createCodeAction(JavaCodeActionContext context, Diagnos
                 context.getParams().isCommandConfigurationUpdateSupported()));
         return codeAction;
     }
+
+    public static CodeAction createCodeAction(JavaCodeActionContext context, Diagnostic diagnostic,
+                                              String quickFixMessage, String participantId) {
+        ExtendedCodeAction codeAction = new ExtendedCodeAction(quickFixMessage);
+        codeAction.setRelevance(0);
+        codeAction.setDiagnostics(Collections.singletonList(diagnostic));
+        codeAction.setKind(CodeActionKind.QuickFix);
+        codeAction.setData(new CodeActionResolveData(context.getUri(), participantId,
+                context.getParams().getRange(), Collections.emptyMap(),
+                context.getParams().isResourceOperationSupported(),
+                context.getParams().isCommandConfigurationUpdateSupported()));
+        return codeAction;
```

Comment:
I meant just calling the other method, so we're not copying code.

Reply:
Done the changes. Missed to remove the duplicate code.